torchmetrics/classification/ranking.py

```diff
@@ -0,0 +1,173 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Optional
+
+import torch
+from torch import Tensor
+
+from torchmetrics.functional.classification.ranking import (
+    _coverage_error_compute,
+    _coverage_error_update,
+    _label_ranking_average_precision_compute,
+    _label_ranking_average_precision_update,
+    _label_ranking_loss_compute,
+    _label_ranking_loss_update,
+)
+from torchmetrics.metric import Metric
+
+
+class CoverageError(Metric):
+    """Computes multilabel coverage error. The score measure how far we need to go through the ranked scores to
```

Comment:
Do we have a reference publication we can put here? So that it is always clear what it actually computes mathematically?